src/modules/auth/resources/ResourcesListContainer.js

```diff
@@ -3,7 +3,7 @@ import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 
 import { DataTable, ButtonWithAuthz } from 'asab-webui';
-import { Container } from 'reactstrap';
+import { Container, Button } from 'reactstrap';
```

Comment:
@petrKavulok You dont use `Button` in the ResourcesListContainer, so it does not have to be loaded